src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -393,6 +393,12 @@
       <arg type="s" name="transaction_address" direction="out"/>
     </method>
 
+    <!-- Experimental generic Search -->
+    <method name="Search">
+      <arg type="as" name="query" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="s" name="context_new" direction="out"/>
```

Comment:
We're sending in multiple search terms (`as` = array of string) but returning a single string (a package name?).  Could use a comment.

Reply:
You're right, it should probably be another array of strings, I was thinking about the returned info as a whole. But I think now it should receive the URLs as an array of strings.